modules/sdk-core/test/unit/dlogproofs.ts

```diff
@@ -0,0 +1,67 @@
+import 'should';
+import * as sinon from 'sinon';
+import {
+  generateNTilde,
+  generateNTildeProof,
+  generateSafePrimes,
+  verifyNtildeProof,
+} from '../../src/account-lib/mpc/tss/ecdsa/rangeproof';
+import { OpenSSL } from '../../src/openssl';
+
+describe('h1H2DiscreteLogProofs', function () {
+  let switchPrime = false;
+  let safePrimeMock: sinon.SinonStub;
+  before(async function () {
+    safePrimeMock = sinon.stub(OpenSSL.prototype, 'generateSafePrime').callsFake(async (bitlength: number) => {
+      if (switchPrime) {
+        switchPrime = false;
+        return BigInt('105026459418240911050597781175405200114409463599422710187772697695413160518507');
+      } else {
+        switchPrime = true;
+        return BigInt('97740038048923029272833872518628089389073263932043585221445032564807403246907');
```

Comment:
I would add a comment above these to show how they where generated `openssl prime -bits 256 -generate -safe`

Reply:
done

Reply:
weird i pushed the change, what's wrong with GH lol

Reply:
nvm your comment was not on the changed line that's why it didnt show outdated